Catappult/appcoins-wallet-android
# [task/APPC-2559] Remove Roles payment endpoints

**What does this PR do?**

   Removes "roles" endpoint to retrieve the store/oem wallet addresses. For the APPC flow, a fair bit of changes had to be done to make it work.

**Database changed?**

   No

**How should this be manually tested?**

  Test every payment method and check if it works correctly, also inject an oem/store id to Trivial Drive and check that it is being used correctly on payments.

**What are the relevant tickets?**

  [APPC-2559](https://aptoide.atlassian.net/browse/APPC-2559)

**Code Review Checklist**
